handlers/meme.js

```diff
@@ -0,0 +1,110 @@
+const _       = require('lodash');
+const rp      = require('request-promise');
+const log4js  = require('log4js');
+
+const LOG = log4js.getLogger('meme');
+
+
+module.exports = {
+  name: 'meme',
+  command: 'meme',
+  handler: meme
+};
+
+// Memegenerator requires an API key as well as the username and password belonging to the key owner
+const MEMEGENERATOR_API_KEY = process.env.MEMEGENERATOR_API_KEY;
+const MEME_USER = process.env.MEME_USER;
+const MEME_PASSWORD = process.env.MEME_PASSWORD;
+const API_URL = 'http://version1.api.memegenerator.net//Instance_Create';
+
+// Language tagging; for browsing purposes on the memegenerator site
+const languageCode = 'en';
+
+// The meme "generator" is actually the background image that is used for the macro. Defaulting to insanity wolf
+let generatorID = 45;
+
+// Default top and bottom text in case we aren't able to pull suitable candidates from chat
+let top_text = 'GOOD';
+let bottom_text = 'SHIT';
+
+function meme(channel, message, params) {
+
+  // Instead of just throwing a console error or timing out of the values aren't set, we can send a message and then quit
+  if(!MEME_USER){
```

Comment:
This formatting stuff will be auto fixed with eslint/jshint

But for if statements, do this

```
if (!MEME_USER) {
  ...
} else if (condition) {
  ...
}
```